archivebox/main.py

```diff
@@ -493,6 +494,13 @@ def status(out_dir: str=OUTPUT_DIR) -> None:
     print(ANSI['black'], '   ...', ANSI['reset'])
 
 
+@enforce_types
+def oneshot(url: str, out_dir: str=OUTPUT_DIR):
```

Comment:
Can you add a docstring to this function explaining how it works in a sentence or two 👍

Reply:
Copied the text from https://github.com/pirate/ArchiveBox/wiki/Roadmap#-archivebox-oneshot